testsuite/forge-cli/src/main.rs

```diff
@@ -525,6 +530,12 @@ fn land_blocking_test_suite(duration: Duration) -> ForgeConfig<'static> {
             } else {
                 60
             })),
+            Some(SystemMetricsThreshold::new(
+                // Threshold of more than 12 CPU cores for 30% of the time
```

Comment:
wording here is very odd.

this is a check that we don't use more than 12 CPU cores for 30% of the time.

(wording suggests that we are checking that we are using at least that much).